app/concepts/matestack/ui/core/form/form.js

```diff
@@ -126,8 +126,11 @@ const componentDef = {
           self.$store.dispatch('navigateTo', {url: path, backwards: false})
           return;
         }
-        self.setProps(self.data, null);
-        self.initValues()
+        if (self.componentConfig["method"] == "post")
```

Comment:
@fiedl this approach is too implicit in my opinion. I think we cannot rely on the correct usage of http methods. A user might want the form not to reset the form even if using the "post" method. (we never know). We need a explicit option for that:
I recommend something like that:

```ruby
        def form_config
          {
            for: @test_model,
            method: :whatever,
            path: "/test_models/#{@test_model.id}",
            success: { emit: "update_successful", reset: false }, # reset == true by default (current behavior)
            failure: { emit: "form_has_errors" } # reset == false by default (current behavior)
          }
        end
```
If you agree on this, please adjust implementation and specs and add the docs (they are missing anyways ;) )

Reply:
I totally agree that the behaviour should be configurable. However, I would like to find a default where the form is not reset for an update action which does not redirect to another page, i.e. shows the same form. Otherwise, when the user clicks "save", the form will be reset the the initial form values (before the changes), which probably would feel like a bug.

I will implement the configuration option as you suggested and give the defaults some further thought before suggesting a solution.

Reply:
Regarding the documentation: I've noticed that the "Parameters" section on https://www.matestack.org/docs/components/form.md is empty, yet. Would you like me to add another example to this page showing the new configuration option, or would you like me to start filling the "Parameters" section with this?

Reply:
@fiedl sorry for the late response. please add the parameter section and add an example for the reset option :)